bigquery_etl/cli/query.py

```diff
@@ -704,9 +704,9 @@ def deploy(ctx, name, sql_dir, project_id):
 
                 if metadata.bigquery and metadata.bigquery.time_partitioning:
                     new_table.time_partitioning = bigquery.TimePartitioning(
-                        metadata.bigquery.time_partitioning.type.bigquery_type(),
-                        metadata.bigquery.time_partitioning.field,
-                        metadata.bigquery.time_partitioning.require_partition_filter,
```

Comment:
The third positional param was getting interpreted as expiration millis, so this was throwing an error.